### PR TITLE
Fix/improve/complete Czech translation

### PR DESCRIPTION
**Pull request by mormegil-cz**
Fixed typos, grammar, and style. Improved typography, added a couple of missing translations.

### CODE REVIEW

**Comment by rugk** [comment]
Please do note though, that you can also use the simple online tool _Crodwin_ for translation. See [out translation guide in the wiki](https://github.com/PrivateBin/PrivateBin/wiki/Translation) for more information. We normally use that and it periodically syncs the translations back to this repository.

File: i18n/cs.json [hunk 1]
```diff
@@ -1,148 +1,148 @@
...
     "Discussion": "Komentáře",
     "Toggle navigation": "Přepnout navigaci",
     "%d seconds": [
-        "%d sekuda",
-        "%d sekundy",
-        "%d sekund",
-        "%d sekund",
-        "%d sekund",
-        "%d sekund"
+        "%d sekunda",
```
[the remark this answers is not in the document]
Comment (mormegil-cz) [reply i18n/cs.json:39]: Well, when I was improving some translations, I automatically inserted a non-breaking space after a “v” preposition as is the norm in Czech (which cannot be solved by CSS; even though my long-term belief is that this _should_ be done automatically by browser layout engines when the content language is cs… never mind). Only afterwards I chose to add the nbps to all the existing translations where needed. (Also, I’m not sure if/where these “%d seconds/minutes/…” translations are used…?) While I don’t see any downside in having nbsps in translations, feel free to replace them with plain spaces, if so desired; it’s not very important in these short messages, anyway.

**Comment by mormegil-cz** [comment]
@rugk: Right. I noticed the merged PRs from Crowdin, and the `crowdin.yml` file, so I tried to find the link to the Crowdin translation project, and failed (there is no “crowdin” text in any of the files in the repository, and crowdin.com leads to a company/product homepage). And having the file translated already at the time, I did not investigate further.

Obviously, when I looked now, I see the translation documentation linked directly from the readme, with direct pointer to Crowdin, etc. Not sure how I missed it the first time. (Even though if you accept translations even this way, Crowdin would not be that much more helpful for me for such smaller changes in the relatively small number of messages.)

**Comment by rugk** [comment]
Yeah it's fine for now. And if you have anything to improve about the doc/how easy it is to find for contributors, then feel free to provide a PR.

File: i18n/cs.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (elrido) [reply i18n/cs.json:39]: All good, just wanted to clarify this use case and give other maintainers the opportunity to speak up.

**Pull request merged**